remoulade/api/schema.py

```diff
@@ -11,3 +11,29 @@ class MessageSchema(Schema):
     kwargs = fields.Dict(allow_none=True)
     options = fields.Dict(allow_none=True)
     delay = fields.Number(validate=validate.Range(min=1), allow_none=True)
+
+
+class PageSchema(Schema):
+    """
+        Class to validate the attributes of a page
+    """
+
+    sort_column = fields.Str(allow_none=True)
+    sort_direction = fields.Str(allow_none=True, validate=validate.OneOf(["asc", "desc"]))
+    size_page = fields.Int(missing=100, validate=validate.Range(min=1))
+    search_keys = fields.List(
```

Comment:
maybe the simplest solution would be store search_keys in remoulade as it should not change often.